app/hub/Views/CreateAccountView/CreateAccountViewContainer.jsx

```diff
@@ -93,9 +95,17 @@ class CreateAccountViewContainer extends Component {
 	}
 
 	/**
-	 * Update input checkbox values by updating state.
+	 * Update legal consetnt checkbox  value by updating state
 	 */
-	_handleCheckboxChange = () => {
+	_handleLegalConsentCheckboxChange = () => {
+		const legalConsentChecked = !this.state.legalConsentChecked;
+		this.setState({ legalConsentChecked });
```

Comment:
Use callback on `setState()` rather than `this.state`.  See [react/no-access-state-in-setstate](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/no-access-state-in-setstate.md)

Reply:
Done